### ISSUE TITLE
web(.html) format size reducing 

### ISSUE THREAD

**Issue by budikesuma** [dipu-bd/lightnovel-crawler#1202]
Can you reduce the size of the `web(.html)` format?  
Since I saw unnecessary multilevel typeface `style` settings. 
I've been trying to edit the `.html` file of the lncrawler result, without detracting from all the features of that typeface `style` setting, and managed to reduce the size of about ⅓ of the previous file.  
I included that file (in `.zip` file below) for you to check. 

My edited typeface `style`:

```
<style>@import url('https://fonts.googleapis.com/css?family=Merriweather:400,400i,700,700i'); html,body{margin:0;padding:0;width:100%;height:100%;position:relative;background-color:#323235;-webkit-font-smoothing:antialiased}

#content{padding:10px 20px;max-width:850px;margin:10px auto;font-size:16px;font-family:'Merriweather', Georgia, serif;text-align:justify;line-height:1.8;border-radius:5px;box-shadow:0 0 10px #000, 0 0 0 1px #000;background-color:#fffff0}

@media (max-width: 925px){#content { margin:5px;max-width:auto} main{min-height:500px;padding:0 10px}}

h1,h2,h3,h4,h5,h6{color:#555;padding:10px;margin:0;text-align:center;line-height:normal}

h1{color:#333336;font-weight:300;margin-bottom:15px}

h1:after{content:'-';margin:10px 30px;height:2px;border-radius:50%;background:#444;display:block;color:transparent}

.link-group{padding:10px;margin:15px 0;display:flex;align-items:center;justify-content:space-between;background:#dde;border:1px solid #dde}

.link-group a{color:#39f;text-decoration:none}

.link-group .btn{color:#333;font-family:sans-serif;font-size:18px;font-weight:Arial, 600;display:inline-block;width:145px;padding:5px;text-align:center;background:#f5f5f5;box-shadow:1px 1px 2px #aac, 0 0 0 1px #ccc;border-radius:5px}

.link-group .btn:hover{background:#ececef}

.link-group .btn:active{box-shadow:1px 1px 2px #cce inset, 0 0 0 1px #ccc}

div#readpos{border-radius:10px;font-weight:bold;font-family:monospace;color:#770;font-size:14px;padding:5px 10px;background:white;box-shadow:0 0 10px #333, 0 0 0 1px #dde;position:fixed;bottom:10px;right:10px;user-select:none;-webkit-user-drag:none}

@media print{#content { margin:0;max-width:none;box-shadow:none}

.link-group{display:none}</style>
```
.
[The_Mech_Touch_Web_html_style_edited.zip](https://github.com/dipu-bd/lightnovel-crawler/files/7568643/The_Mech_Touch_Web_mod.zip)

**Comment by dipu-bd** [comment]
you can create a pull request with your changes.

**Comment by dipu-bd** [comment]
@budikesuma I don't know why you closed this issue, was this solved?

**Comment by budikesuma** [comment]
> @budikesuma I don't know why you closed this issue, was this solved?

Nope, wasn't solved yet. 

previously I posted without looking at your design style code on github, but after I looked it up, it's almost the same, maybe python will reproduce it into nested iterative codes?  and the style becomes larger by about 10 KB.  
Since there is no difference between this code style and yours, then my thread becomes useless...
